src/ensembl/src/shared/accordion/components/AccordionItemState.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { DivAttributes } from '../helpers/types';
+import { Consumer as ItemConsumer, ItemContext } from './ItemContext';
+
+type Props = Pick<DivAttributes, Exclude<keyof DivAttributes, 'children'>> & {
+  children(
+    args: Partial<{ expanded: boolean; disabled: boolean }>
+  ): React.ReactNode;
+};
+
+const AccordionItemState = (props: Props) => {
+  const renderChildren = (itemContext: ItemContext): JSX.Element => {
```

Comment:
This function also probably doesn't even need to exits. Also, props.children probably don't need to be wrapped in a fragment.

Reply:
I've remove the function. The fragment is needed here as ItemContext expects only one child.